Sources/Zephyr.swift

```diff
@@ -11,6 +11,10 @@ import Foundation
 import UIKit
 #endif
 
+#if os(watchOS)
```

Comment:
can this be `#elif`

Reply:
Sure! I added this change in #[8b9248d](https://github.com/ArtSabintsev/Zephyr/pull/60/commits/8b9248d77987af7cd0df56b414760ea9006237dc)